### PR TITLE
(feature)[Frontend] Create register view

### PR DESCRIPTION
**Pull request by VictoriaRSsousa**

### CODE REVIEW

File: src/services/rest/pet.rest.ts [hunk 1]
```diff
@@ -1,27 +1,27 @@
-import type { Observable } from 'rxjs'
-import api from "../api-config/rxjs";
-import type Pet from '@/models/pet.model'
-export class PetRest {
+// import type { Observable } from 'rxjs'
+// import api from "../api-config/rxjs";
+// import type Pet from '@/models/pet.model'
+// export class PetRest {
```
Comment (eltonls) [inline src/services/rest/pet.rest.ts:5]: Delete comments!

File: src/services/rest/pet.rest.ts [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
+//     }
+//     updatePet(pet:Pet): Observable<any>{
+//         const url:string = `/`
+//         return api.put(url,pet)
+//     }
+//     deletePet(id:number): Observable<any>{
+//         const url:string = `/${id}`
+//         return api.deleteR(url)
+//     }
+// }
```
Comment (eltonls) [inline src/services/rest/pet.rest.ts:27]: Delete comments

File: src/views/Home/home.service.ts [hunk 3]
```diff
@@ -1,19 +1,19 @@
-import { PetRest } from "@/services/rest/pet.rest";
+//import { PetRest } from "@/services/rest/pet.rest";
 import { BehaviorSubject, Observable, Subject } from "rxjs";
 
 export class HomeService {
   constructor(
-    private _petRest = new PetRest()
+    //private _petRest = new PetRest()
```
Comment (eltonls) [inline src/views/Home/home.service.ts:6]: delete comments

File: src/views/Home/home.service.ts [hunk 4]
```diff
@@ -1,19 +1,19 @@
-import { PetRest } from "@/services/rest/pet.rest";
+//import { PetRest } from "@/services/rest/pet.rest";
```
Comment (eltonls) [inline src/views/Home/home.service.ts:1]: delete comments

File: src/views/Home/home.service.ts [hunk 5]
```diff
@@ -1,19 +1,19 @@
...
-      }
-    })
-  }
+  // getAllPets(): void {
+  //   this._petRest.getAllPets().pipe().subscribe({
+  //     next: (response) => {
+  //       this.pets$.next(response)
+  //     }
+  //   })
+  // }
```
Comment (eltonls) [inline src/views/Home/home.service.ts:18]: delete comments

File: src/views/Home/home.vue [hunk 6]
```diff
@@ -5,25 +5,25 @@ export default{
...
-      });
+      // this.service.pets.pipe().subscribe({
+      //   next: (response) => {
+      //     this.pets = response.data;
+      //     this.total = this.pets.length;
+      //     setTimeout(() => {
+      //       this.isLoading = false;
+      //     }, 700);
+      //   }
+      // });
```
Comment (eltonls) [inline src/views/Home/home.vue:24]: delete comments

**Review by HelenAndrade — APPROVED** [review]

**Review by VictorManoel-Timbo — APPROVED** [review]

**Review by eltonls — APPROVED** [review]

**Pull request merged**